scripts/hostmap-crtsh.nse

```diff
@@ -88,31 +96,52 @@ local function is_valid_hostname (name)
   return true
 end
 
+-- TRUE subdomain check (ends with ".parent")
+local function is_subdomain(name, parent)
+  name = name:lower()
```

Comment:
Good choice generally, but you should not have to do this here. You know that `name` is already lowercase because of the iterator in `query_ctlogs()`:
```lua
for _, name in ipairs(stringaux.strsplit("%s+", names:lower())) do
```